### ISSUE TITLE
changing znc username in network settings also changes irc nick

### ISSUE THREAD

**Issue by LaMaGit** [kiwiirc/kiwiirc#1694]
Changing the username or server password (containing a <username>: part) within ZNC settings also changes the irc nickname in network settings.

**Comment by ItsOnlyBinary** [comment]
@LaMaGit would be awesome if you could test this pull-request solves your issues completely.

There is pre-built client files here: https://builds.kiwiirc.com/zips/kiwiirc_pull-1695.zip

**Comment by LaMaGit** [comment]
Overwriting my static kiwiirc files with the provided ones seems to fix my nickname issues. Nickname is no longer replaced by my ZNC username. When connecting to ZNC the configured nickname is now correctly replaced with the one provided by the bouncer. ZNC client identifiers work well.
Replacing my static files with the ones provided prevents the usage of '"startupScreen": "personal",'

**Comment by ItsOnlyBinary** [comment]
What happens when you try to use personal startup?

**Comment by LaMaGit** [comment]
default welcome page comes up

**Comment by ItsOnlyBinary** [comment]
that maybe your browser caching the config.json, seems to work ok here

**Comment by LaMaGit** [comment]
I found my error. I used https://github.com/kiwiirc/kiwiirc/releases/download/v1.6.1/kiwiirc-server_v1.6.1-1_linux_armhf.deb to install kiwiirc. This uses /etc/kiwiirc/client.json. The files you provided use the config.json file inside the kiwiirc static folder to configure the client (i did not customize the correct file. the .deb package replaces the file inside the static folder with a symlink to /etc/kiwiirc/client.json)
